src/migration/event-migration-footer.component.html

```diff
@@ -3,5 +3,6 @@
        ng-click="migration.completeMigration()"
        role="button"
        ng-class="{disabled: !migration.migrationReady()}"
-       ng-bind="::migration.destination.description"></a>
-</div>
\ No newline at end of file
+       ng-bind="::migration.destination.description"
```

Comment:
Do we still need the `ng-bind`? The documentation says that `ng-bind` should set the text content of the element: https://docs.angularjs.org/api/ng/directive/ngBind

> The ngBind attribute tells AngularJS to replace the text content of the specified HTML element with the value of a given expression, and to update the text content when the value of that expression changes.

So either we don't need it anymore, or instead of adding the `translate-once` we should make sure the `migration.destination.description` has a value?

Sorry to revisit this after asking to make the text translatable, only realized this now.

Reply:
Is getting overwritten by the description indeed 
![Schermafbeelding 2020-10-23 om 14 16 45](https://user-images.githubusercontent.com/66943525/97002598-7c24b900-153a-11eb-9929-2f9c444e915a.png)

So I'll use the translation when the description is undefined?

Reply:
![Schermafbeelding 2020-10-23 om 14 19 21](https://user-images.githubusercontent.com/66943525/97002792-c6a63580-153a-11eb-9c9e-862b0f04288b.png)

Reply:
Thanks! 👍